woeip/apps/air_quality/tests/test_dustrak.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=W0621
```

Comment:
This is suppressing warnings for use of the ```target_data``` name. For lines 25 and 44, it makes sense to suppress the warning; pylint fails to recognize they are function calls. However, I am less sure about lines 17-22. There, ```target_data``` is defined as a variable within the ```target_data``` function. Would it be better to rename ```target_data``` in lines 17-22 to ```target_output```, rather than simply suppress the warning? Or does pytest.fixtures rely on the logic of calling the function within itself?

Reply:
It took me forever to find a [good list of error names](https://github.com/janjur/readable-pylint-messages/blob/master/README.md). Do you just ...know them?

Reply:
Nope. When you run pylint the errors are printed in parentheses.

Reply:
Hmm, all I saw were the messages and not the names. Guess I know what to look for next time.